assets/src/hooks/v2/use_api_response.tsx

```diff
@@ -57,7 +57,7 @@ const doFailureBuffer = (
       // This will trigger until a success API response is received.
       setApiResponse((prevApiResponse) => {
         if (prevApiResponse != null && prevApiResponse.state === "success") {
-          Sentry.captureMessage("Entering no-data state.");
+          Raven.captureMessage("Entering no-data state.", {level: "info"});
```

Comment:
Oh! This is neat! I completely missed that we can set the log level like that. This is really helpful. I assumed there was no way to stop Sentry from freaking out about every error/message we sent to it.